imports/wasi_snapshot_preview1/usage_test.go

```diff
@@ -16,18 +16,17 @@ var wasiArg []byte
 
 func TestInstantiateModule(t *testing.T) {
 	r := wazero.NewRuntime(testCtx)
+	defer r.Close(testCtx)
 
 	stdout := bytes.NewBuffer(nil)
 
 	// Configure WASI to write stdout to a buffer, so that we can verify it later.
 	sys := wazero.NewModuleConfig().WithStdout(stdout)
-	wm, err := Instantiate(testCtx, r)
+	_, err := Instantiate(testCtx, r)
```

Comment:
While cleaning, can consider switching to `MustInstantiate`

Reply:
good idea